lib/rototiller/task/collections/command_collection.rb

```diff
@@ -0,0 +1,14 @@
+require 'rototiller/task/collections/param_collection'
+require 'rototiller/task/params/command'
+
+module Rototiller
+  module Task
+
+    class CommandCollection < ParamCollection
+      def allowed_class
+        Command
+      end
```

Comment:
i actually thought i could just infer the class name here, but it was kludgy by finding `.class` and then removing `'Collection'`  :-(
that would have made it so we didn't need the children